src/main/java/com/theoryinpractise/googleformatter/GoogleFormatterMojo.java

```diff
@@ -119,6 +119,7 @@ public void execute() throws MojoExecutionException {
   }
 
   private Set<File> filterUnchangedFiles(Set<File> originalFiles) throws MojoExecutionException {
+    MavenProject project = session.getTopLevelProject();
```

Comment:
This variable shadows the `project` field. Maybe rename it to `topLevelProject`?

Reply:
I wanted to minimize the amount of changes

Reply:
done